### PR TITLE
Rework links to old tutorials

### PR DESCRIPTION
**Pull request by MalteEbner**
## Important referrals:
Sometimes we cannot do anything about the links -> no action.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -326,10 +328,11 @@ pylint lightly/core.py
   we need to make training deep learning models more data efficient to achieve widespread adoption. One step to achieve this goal is by leveraging self-supervised learning. The company behind lightly commited to keep this framework open-source.
 
 - If this framework is free, how is the company behind lightly making money?
-  - Training self-supervised models is only one part of our solution. [The company behind lightly](https://lightly.ai/) focuses on processing and analyzing embeddings created by self-supervised models. 
-  By building, what we call a self-supervised active learning loop we help companies understand and work with their data more efficiently. This framework acts as an interface
-  for our platform to easily upload and download datasets, embeddings and models. Whereas 
-  the platform will cost for additional features this frameworks will always remain free of charge (even for commercial use).
+  - Training self-supervised models is only one part of our solution.
```
Comment (japrescott) [inline README.md:331]: weird indentation

File: README.md [hunk 2]
```diff
@@ -326,10 +328,11 @@ pylint lightly/core.py
   we need to make training deep learning models more data efficient to achieve widespread adoption. One step to achieve this goal is by leveraging self-supervised learning. The company behind lightly commited to keep this framework open-source.
 
 - If this framework is free, how is the company behind lightly making money?
-  - Training self-supervised models is only one part of our solution. [The company behind lightly](https://lightly.ai/) focuses on processing and analyzing embeddings created by self-supervised models. 
-  By building, what we call a self-supervised active learning loop we help companies understand and work with their data more efficiently. This framework acts as an interface
-  for our platform to easily upload and download datasets, embeddings and models. Whereas 
-  the platform will cost for additional features this frameworks will always remain free of charge (even for commercial use).
+  - Training self-supervised models is only one part of our solution. 
+  [The company behind lightly](https://lightly.ai/) focuses on processing and analyzing embeddings created by self-supervised models.
```
Comment (japrescott) [inline README.md:332]: Lightly

File: README.md [hunk 3]
```diff
@@ -326,10 +328,11 @@ pylint lightly/core.py
...
-  - Training self-supervised models is only one part of our solution. [The company behind lightly](https://lightly.ai/) focuses on processing and analyzing embeddings created by self-supervised models. 
-  By building, what we call a self-supervised active learning loop we help companies understand and work with their data more efficiently. This framework acts as an interface
-  for our platform to easily upload and download datasets, embeddings and models. Whereas 
-  the platform will cost for additional features this frameworks will always remain free of charge (even for commercial use).
+  - Training self-supervised models is only one part of our solution. 
+  [The company behind lightly](https://lightly.ai/) focuses on processing and analyzing embeddings created by self-supervised models. 
+  By building, what we call a self-supervised active learning loop we help companies understand and work with their data more efficiently. 
+  As the [Lightly Solution](https://docs.lightly.ai) is a freemium product, you can try it out for free. However, we will charge for some features.
+  - In any case this framework will always be free to use, even for commercial purposes.
```
Comment (japrescott) [inline README.md:335]: weird indentation

**Review by japrescott — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -322,14 +324,15 @@ pylint lightly/core.py
   - Create an issue if you encounter bugs or have ideas for features we should implement. You can also add your own code by forking this repository and creating a PR. More details about how to contribute with code is in our [contribution guide](CONTRIBUTING.md).
 
 - Is this framework for free?
-  - Yes, this framework completely free to use and we provide the code. We believe that
-  we need to make training deep learning models more data efficient to achieve widespread adoption. One step to achieve this goal is by leveraging self-supervised learning. The company behind lightly commited to keep this framework open-source.
+  - Yes, this framework is completely free to use and we provide the source code. We believe that
+  we need to make training deep learning models more data efficient to achieve widespread adoption. One step to achieve this goal is by leveraging self-supervised learning. The company behind Lightly is committed to keep this framework open-source.
```
Comment (MalteEbner) [inline README.md:328]: These changes are unrelated to the issue, I just spotted them.

File: README.md [hunk 5]
```diff
@@ -61,10 +61,12 @@ Want to jump to the tutorials and see lightly in action?
 - [Use lightly with custom augmentations](https://docs.lightly.ai/self-supervised-learning/tutorials/package/tutorial_custom_augmentations.html)
 - [Pre-train a Detectron2 Backbone with Lightly](https://docs.lightly.ai/self-supervised-learning/tutorials/package/tutorial_pretrain_detectron2.html)
 
-Tutorials of using the lightly packge together with the Lightly Platform:
+Documentation and tutorials of using the Lightly solution:
 
+- [General Docs of Lightly solution](https://docs.lightly.ai)
 - [Active Learning Using YOLOv7 and Comma10k](https://docs.lightly.ai/docs/active-learning-yolov7)
-- [Active Learning with the Nvidia TLT](https://github.com/lightly-ai/NvidiaTLTActiveLearning)
+- many more tutorials
```
[the remark this answers is not in the document]
Comment (japrescott) [reply README.md:68]: I agree, but at the very least we can mention 4-5 here anyway otherwise it looks very empty or as if there is nothing available

[the remark this answers is not in the document]
Comment (MalteEbner) [reply README.md]: I added more: to all except one, so that I can write `and more ...` ;)

**Review by japrescott — APPROVED** [review]
